[docs] Fix misleading and add info about old versions (#7290)
deckhouse/deckhouse: docs/documentation/pages/DECKHOUSE-OVERVIEW.md

permalink: en/deckhouse-overview.html
---

> Welcome to the home page of the Deckhouse Kubernetes Platform cluster management platform documentation! We recommend starting with the [Getting started]({% if site.mode == 'local' %}{{ site.urls[page.lang] }}{% endif %}/gs/) guide for those who haven't yet tried the platform. It provides step-by-step instructions for deploying the platform to any infrastructure.

Reviewer: ```suggestion
Welcome to the home page of the Deckhouse Kubernetes Platform documentation! We recommend starting with the [Getting started]({% if site.mode == 'local' %}{{ site.urls[page.lang] }}{% endif %}/gs/) guide for those who haven't yet tried the platform. It provides step-by-step instructions for deploying the platform to any infrastructure.
```

Here are some tips on how to find what you need:

> - Note that the Deckhouse Kubernetes Platform documentation for different versions may vary. Select the version you need from the drop-down list at the top of the page. The list contains up to date versions of the documentation.

Reviewer: ```suggestion
- Note that the Deckhouse Kubernetes Platform documentation for different versions may vary. Select the version you need from the drop-down list at the top of the page. The list contains up-to-date versions of the documentation.
```

> - Note that the Deckhouse documentation for different versions may vary. Select the version you need from the drop-down list at the top of the page. The list contains up to date versions of the Deckhouse documentation.

Reviewer: ```suggestion
- Note that the Deckhouse Kubernetes Platform documentation for different versions may vary. Select the version you need from the drop-down list at the top of the page. The list contains up to date versions of the Deckhouse documentation.
```

> Welcome to the home page of the Deckhouse Kubernetes cluster management platform documentation! We recommend starting with the [Getting started]({% if site.mode == 'local' %}{{ site.urls[page.lang] }}{% endif %}/gs/) guide for those who haven't yet tried the platform. It provides step-by-step instructions for deploying the platform to any infrastructure.

Reviewer 2: ```suggestion
Welcome to the home page of the Deckhouse Kubernetes Platform cluster management platform documentation! We recommend starting with the [Getting started]({% if site.mode == 'local' %}{{ site.urls[page.lang] }}{% endif %}/gs/) guide for those who haven't yet tried the platform. It provides step-by-step instructions for deploying the platform to any infrastructure.
```

Here are some tips on how to find what you need:
- Note that the documentation is version-specific and may differ from version to version. You can select the Deckhouse version in the drop-down list at the top left of the page. Please note that the list contains documentation versions for current versions of Deckhouse. For example, to access this document of the Deckhouse version `v1.49`, it is enough to access the URL `https://deckhouse.ru/documentation/v1.49/deckhouse-overview.html`.
- If you know what you're looking for exactly, use the search box at the top of the page.
- Check [this list](revision-comparison.html) if you are looking for information on a particular module.
- The menu on the left is for searching by scope. Try to guess which section might contain what you are looking for.

> Если возникнут вопросы, вы можете обратиться за помощью в наш [тelegram-канал]({{ site.social_links[page.lang]['telegram'] }}). Мы обязательно поможем и проконсультируем.

Reviewer 3: ```suggestion
If you have any questions, feel free to contact us at [Telegram channel]({{ site.social_links[page.lang]['telegram'] }}). We will do our best to help you.
```

> Если вы используете Enterprise версию, можете написать нам [на почту](mailto:support@deckhouse.ru), мы также окажем вам поддержку.

Reviewer 3: ```suggestion
Enterprise Edition users can contact us at [email](mailto:support@deckhouse.ru). We will be glad to provide assistance!
```

Хотите улучшить Deckhouse? Можете завести [задачу](https://github.com/deckhouse/deckhouse/issues/), предложить свою [идею](https://github.com/deckhouse/deckhouse/discussions) или [решение](https://github.com/deckhouse/deckhouse/blob/main/CONTRIBUTING.md) на GitHub.

> А если вам хочется большего, присоединяйтесь к нашей [команде](https://job.flant.ru/)! Мы рады новым специалистам.

Reviewer 2: Текст для перевода

> Хотите улучшить Deckhouse? Можете завести [задачу](https://github.com/deckhouse/deckhouse/issues/), предложить свою [идею](https://github.com/deckhouse/deckhouse/discussions) или [решение](https://github.com/deckhouse/deckhouse/blob/main/CONTRIBUTING.md) на GitHub.
> 
> А если вам хочется большего, присоединяйтесь к нашей [команде](https://job.flant.ru/)! Мы рады новым специалистам.

Reviewer 3: ```suggestion
Interested in improving Deckhouse? Feel free to open a GitHub [Issue](https://github.com/deckhouse/deckhouse/issues/), suggest an [idea](https://github.com/deckhouse/deckhouse/discussions) or join the list of [contributors](https://github.com/deckhouse/deckhouse/blob/main/CONTRIBUTING.md).

Feel like you can contribute a lot more? Then join our [team](https://job.flant.ru/)! We look forward to welcoming new professionals!
```

Welcome to the home page of the Deckhouse Kubernetes cluster management platform documentation! We recommend starting with the [Getting started]({% if site.mode == 'local' %}{{ site.urls[page.lang] }}{% endif %}/gs/) guide for those who haven't yet tried the platform. It provides step-by-step instructions for deploying the platform to any infrastructure.

Here are some tips on how to find what you need:

> - Note that the documentation is version-specific and may differ from version to version. You can select the Deckhouse version in the drop-down list at the top left of the page. Please note that the list contains documentation versions for current versions of Deckhouse. For example, to access this document of the Deckhouse version `v1.49`, it is enough to access the URL `https://deckhouse.ru/documentation/v1.49/deckhouse-overview.html`.

Reviewer: ```suggestion
- Документация по Deckhouse разных версий может отличаться. Выберите нужную версию в выпадающем списке вверху страницы. В списке доступны актуальные версии документации по Deckhouse.
```

@kirkonru Please...

> - If you know what you're looking for exactly, use the search box at the top of the page.
> - Check [this list](revision-comparison.html) if you are looking for information on a particular module.
> - The menu on the left is for searching by scope. Try to guess which section might contain what you are looking for.

Reviewer 2: ```suggestion
- If you know what you are looking for, use the search. 
- If you need a specific module, find it in the [list](revision-comparison.html ).
- To search by application, use the menu.
```

Reviewer: This is @kirkonru 's area

Reviewer 2: Текст для перевода
